src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,26 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## March 9, 2023
+
+### AEP React Native libraries 2.0.0
```

Comment:
Can we remove 2.0.0 from the title as the version numbers differ for each extension? How about `AEP React Native libraries` or `AEP React Native library updates to support MobileCore 2.x for Android and compatible extensions`

Reply:
Since AEP is against branding guidelines, I'd like to **not** include that in the title, if possible. 

Since we're already in the current versions repo, can't we just have the title as "React Native libraries" and then have the subsequent blurb explain that it's for Mobile Core 2.x?

Reply:
@praveek  how about `AEP React Native library updates` ?

Reply:
@calebk1  we have ACP and AEP React Native libraries, we can't just say `React Native libraries`.

Reply:
or simply `AEP React Native plugins` since we cover the other details in the description below

Reply:
https://reactnative.dev/docs/libraries @emdobrin  In React Native world, we don't call them plugins.

Reply:
If that is the case we were referring to them incorrectly in previous release notes too? Can we make them consistent, maybe use wrapper instead?

Reply:
The title is updated to `AEP React Native libraries` for now. Let me know if you have any other options.

Reply:
@emdobrin `wrapper` is more of an internal concept. It could be very confusing for React Native developers.